app/helpers/Evaluation/ScoreCalculators/IScoreCalculator.php

```diff
@@ -24,6 +24,15 @@ public function computeScore(string $scoreConfig, array $testResults): float;
    */
   public function isScoreConfigValid(string $scoreConfig): bool;
 
+  /**
+   * Performs validation and normalization on config string.
+   * This should be used instead of validation when the score config is processed as API input.
```

Comment:
If this method should be used instead of `isScoreConfigValid` why not use it right now, it is used only two times...